hack/local-up-cluster.sh

```diff
@@ -115,7 +115,6 @@ sudo "${GO_OUT}/kubelet" \
   --hostname_override="127.0.0.1" \
   --address="127.0.0.1" \
   --api_servers="${API_HOST}:${API_PORT}" \
-  --auth_path="${KUBE_ROOT}/hack/.test-cmd-auth" \
```

Comment:
We're trying to move towards https by default, so rather than removing the auth_path argument, it's probably preferable to actually build the contents of the file correctly.  That would allow the local environment to serve securely and be properly validated by the kubelet.

Since some of the e2e providers already verify the the https connection, there is every expectation that this would work properly.